examples/context/motivation-solution.js

```diff
@@ -1,26 +1,26 @@
 // highlight-range{1-4}
-// Context lets us pass a value deep into the component tree
-// without explicitly threading it through every component.
-// Create a context for the current theme (with "light" as the default).
-const ThemeContext = React.createContext('light');
+// Le Contexte nous permet de transmettre une prop profondément dans l'arborescence du composant
+// sans la faire passer explicitement à travers tous les composants.
+// Créez un contexte pour le thème (avec « clair » par défaut).
```

Comment:
```suggestion
// Crée un contexte pour le thème (avec “light” comme valeur par défaut).
```

Outre qu'on ne traduit pas les valeurs internes, quand on cite de l'anglais, on utilise les guillemets anglais